.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: cpp
 sudo: false
 
+notifications:
+    slack: ringmesh:5cOVjJEAHsS8HkmRQxoTWl0f
+    on_success: change
+    on_failure: always
```

Comment:
I do not know if this won't sent to much notifications. If you consider the number of failure on "in progress" branches.